ui/rollSection/rollSection.scss

```diff
@@ -19,7 +17,7 @@
   padding: 1px 2px;
   border-width: 1px;
   border-radius: 3px;
-  background-color: $primary-background-color;
+  @include theme-background-color(primary-background-color);
```

Comment:
Most conversions are just adding `@include theme-` and using the same color name.